src/script/lua_api/l_particles_local.cpp

```diff
@@ -0,0 +1,205 @@
+/*
```

Comment:
I could but that file includes a lot of backwards compatibility related stuff that isn't really wanted in a new api.